ChangeLog.d/rm-ssl-conf-curves.txt

```diff
@@ -0,0 +1,4 @@
+Removals
+   * Remove the function mbedtls_ssl_conf_curves() which had been deprecated
```

Comment:
Please update `3.0-migration-guide.md` here as well. Unless we just remove it at this point?